cypress/integration/project/patches.ts

```diff
@@ -16,20 +13,4 @@ describe("Project Patches Page", () => {
     cy.dataCy("project-patches-link").first().click();
     cy.location("pathname").should("eq", route);
   });
-
-  it("Page title and data rows are displayed.", () => {
```

Comment:
Should we replace this with something more generic?

Reply:
I capture its functionality in the previous test